modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/slot/SlotManagerClusterMode.java

```diff
@@ -145,10 +145,10 @@ public Slot getSlot(String queueName, String nodeId) {
         String lockKey = queueName + SlotManagerClusterMode.class;
 
         synchronized (lockKey.intern()) {
-            slotToBeAssigned = getOverlappedSlot(nodeId, queueName);
+            slotToBeAssigned = getUnassignedSlot(queueName);
```

Comment:
We thought that unassigned slots should be given priory since they are the oldest messages